src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/controller/RefreshControllerIntegrationTest.java

```diff
@@ -16,14 +16,10 @@
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
-import org.springframework.util.LinkedMultiValueMap;
```

Comment:
This whole file can be removed as all tests in `RefreshControllerIntegrationTest` are replicated in `RefreshControllerIntegrationV2Test` for V2.

Reply:
RefreshControllerIntegrationV2Test i think has been renamed to RefreshControllerRefreshJobIntegrationTest as part of review comments

Reply:
but the files look to contain different tests, please take another tlook

Reply:
After discussing with Matt I agree that the test class can be removed, and it has been. Done